pyproject.toml

```diff
@@ -9,10 +9,11 @@ python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
 requests = "^2.25.1"
-
-[tool.poetry.dev-dependencies]
+gunicorn = "^20.1.0"
 pytest = "^6.2.4"
 selenium = "^3.141.0"
+
+[tool.poetry.dev-dependencies]
```

Comment:
Is it intentional that you've made pytest/selenium non-dev dependencies?

Reply:
Yeah, because pytest could not be installed when I tried running it as a dev-dependency in docker. Eventually, I abandoned doing testing.